tests/hats_import/verification/test_run_verification.py

```diff
@@ -50,11 +50,11 @@ def test_test_file_sets(small_sky_object_catalog, wrong_files_and_rows_dir, tmp_
     verifier = runner.Verifier.from_args(args)
     passed = verifier.test_file_sets()
     assert not passed, "bad catalog passed"
-    bad_files = {
-        str(COMMON_DIR / "Npix=11.extra_file.parquet"),
-        str(COMMON_DIR / "Npix=11.missing_file.parquet"),
+    expected_bad_file_names = {"Npix=11.extra_file.parquet", "Npix=11.missing_file.parquet"}
+    actual_bad_file_names = {
+        file_name.replace("\\", "/").split("/")[-1] for file_name in verifier.results_df.bad_files.squeeze()
```

Comment:
I think you can use `os.path.basename` here

Reply:
The execution OS might not match the path OS. I discovered this issue when adding a test under hats-cloudtests.